conformance/utils/http/http.go

```diff
@@ -44,9 +45,15 @@ type ExpectedRequest struct {
 	Headers map[string]string
 }
 
-// MakeRequestAndExpectResponse makes a request with the given parameters and
-// verifies the response matches the provided ExpectedResponse.
-func MakeRequestAndExpectResponse(t *testing.T, r roundtripper.RoundTripper, gwAddr string, expected ExpectedResponse) {
+const maxConsistencyPeriodPerRequest = 60 * time.Second
```

Comment:
As I understand it, these tests require 3 consecutive successful requests to have occurred within this time period, and this time starts once a Route transitions to a status of accepted. If that's all correct, I think I'd prefer starting with a lower value of 30 seconds + renaming this to something like `maxTimeToConsistency`. Not sure on naming, but the `PerRequest` part throws me off in the current name.

Reply:
The route transition happens outside the scope of the functions I've modified/added. The timer starts when the function is called essentially. The caller has already verified the status transition before calling `MakeRequestAndExpectEventuallyConsistentResponse`